test/Microsoft.ML.Benchmarks/Helpers/DataPathProvider.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+
+namespace Microsoft.ML.Benchmarks
+{
+    internal class DataPathProvider
+    {
+        private static string RootPath = Directory.GetParent(Directory.GetCurrentDirectory()).Parent.Parent.Parent.Parent.Parent.Parent.Parent.FullName;
```

Comment:
Thats a lot of parents. Could this be a maintenance hassle if we change our directory structure? 

What is the runtime directory for this?

Reply:
we are already using the similar structure
https://github.com/dotnet/machinelearning/blob/master/test/Microsoft.ML.TestFramework/BaseTestClass.cs#L40

yes if we change build structure then we will have to change everything.

The current Runtime Directory is 
```C#
C:\git\Mlnet\bin\AnyCPU.Release-netfx\Microsoft.ML.Benchmarks.Tests\netcoreapp2.1\
```